docs/docs/contact-us/index.md

```diff
@@ -0,0 +1,19 @@
+---
+authors: Fragcolor & contributors
+license: CC-BY-SA-4.0
+---
+
+![](assets/ContactUsLogo.png)
+
+# Contact Us
+
+Do you have a burning question? A bug to report? Or perhaps some feedback that you would like to give us? 
+
+Join us in our [Discord](https://discord.fragnova.com/) server and other social media platforms to keep in contact with us and receive news on the latest updates. You can connect with other like-minded Fragnovans too!
+
+| | | |
+| :----------: | :----------: |:----------: |
+|[![Join us on Discord!](assets/fontawesome/discord.svg){: style="height:100px;width:100px"}](https://discord.fragnova.com/) <br> <span style="font-weight:600;font-size:20px">[Discord](https://discord.fragnova.com/)</span> | [![Follow us on Twitter!](assets/fontawesome//twitter.svg){: style="height:100px;width:100px"}](https://twitter.com/fragnova)<br> <span style="font-weight:600;font-size:20px">[Twitter](https://twitter.com/fragnova)</span> | [![Follow us on Telegram!](assets/fontawesome/telegram.svg){: style="height:100px;width:100px"}](https://t.me/fragnova)<br> <span style="font-weight:600;font-size:20px">[Twitter](https://t.me/fragnova)</span> |
+| | | |
```

Comment:
What's the point of this section? We already have all those SN links at the bottom of every page (see the footer).